src/export/pdfExporter.ts

```diff
@@ -0,0 +1,81 @@
+import { appendFileSync } from "fs";
+import path = require('path');
+import jsPDF from "jspdf";
+import autoTable from "jspdf-autotable";
+import { PackageInfo } from '../model/packageInfo';
+import { Exporter } from './exporter';
+
+export class PDFExporter implements Exporter {
+    exportBom(packageInfos: PackageInfo[]): void {
+        this.exportToPDF(packageInfos);
+    }
+
+
+    exportToPDF(packageInfos: PackageInfo[]) {
```

Comment:
Is there a reason why you put this in a separate function?

Reply:
No there was no special reason for that, I can implement everything inside of export(...) function.